justfile

```diff
@@ -60,7 +60,21 @@ _fmt := if env_var_or_default("GITHUB_ACTIONS", "") != "true" { "" } else {
     ```
 }
 
-_features := if features != "" { "--no-default-features --features=" + features } else { "" }
+_features := if features == "all" {
+        "--all-features"
+    } else if features != "" {
+        "--no-default-features --features=" + features
+    } else { "" }
+
+
+# Use nextest if it's available.
+_test := ```
+        if command -v cargo-nextest >/dev/null 2>&1; then
```

Comment:
nit, take it or leave it: you can also detect if a cargo subcommand is available using `cargo --list`; this _might_ be more robust than checking if a `cargo-nextest` executable is on the `PATH`?